src/test/java/net/datafaker/providers/base/AbstractBasicProviderTest.java

```diff
@@ -1,7 +1,5 @@
 package net.datafaker.providers.base;
 
-
-
 import org.junit.jupiter.api.TestInstance;
 
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
I would suggest to remove this class completely

I moved logic to `BaseFakerTest` so it should work

Reply:
👍